testing/common_testing_util.py

```diff
@@ -22,18 +23,25 @@
 
 path_repo_root: Path = Path(__file__).parents[FOLDER_LEVELS_FROM_ROOT].absolute()
 
-path_temp: str = str(path_repo_root / "tmp")
+if not sys.platform.startswith('win'):
+    path_temp: Path = Path("/tmp")
+else:
+    path_temp: Path = Path(Path.home().parts[0], 'temp')
+
+timestr = time.strftime("%Y%m%d-%H%M%S")
+tmptimestr = "tmp" + timestr
+new_path_temp = path_temp / tmptimestr
```

Comment:
I think it is a good idea to add time stamp to this, as it prevents accidentally reusing older data.